flask_ades_wpst/ades_base.py

```diff
@@ -31,8 +22,19 @@ def __init__(self, app_config):
             # platform here, you must also add it to the valid_platforms
             # tuple default argument to the flask_wpst function in
             # flask_wpst.py.
-            raise ValueError("Platform {} not implemented.".format(self._platform))
-        self._ades = ADES_Platform()
+            raise ValueError("Platform {} not implemented.".
+                             format(self._platform))
+        self._default_user = "anonymous"
+        self._ades_id = app_config["ADES_ID"]
+        self._ades = ADES_Platform(self._ades_id)
+        ades_home_dir = os.path.join("./ades", self._ades_id)
+        if not os.path.isdir(ades_home_dir):
+            os.mkdir(ades_home_dir)
```

Comment:
@jjacob7734: can you use https://docs.python.org/3/library/os.html#os.makedirs instead? The line will croak unless I manually create the `./ades` directory.

Reply:
Good suggestion.  Done.  Please test on ADES-K8s and if all is well, we can merge.